runtime/onert/core/src/compiler/OperationValidator.cc

```diff
@@ -519,6 +519,21 @@ void OperationValidator::visit(const ir::operation::Floor &node)
   OP_REQUIRES(_ctx.at(output_index).shape() == _ctx.at(input_index).shape());
 }
 
+void OperationValidator::visit(const ir::operation::FusedBatchNorm &node)
```

Comment:
You don't need to implement `OperationValidator` if there is nothing to check

Reply:
How should I determine if I need to check?
I don't know the standard.

Reply:
There is no standard because this is different on each operation. Maybe type checking?